openapi/smart-events.yaml

```diff
@@ -974,7 +989,28 @@ components:
         transformationTemplate:
           type: string
         action:
-          $ref: '#/components/schemas/BaseAction'
+          $ref: '#/components/schemas/Action'
+        source:
+          $ref: '#/components/schemas/Source'
+    ProcessorType:
+      enum:
+      - SOURCE
+      - SINK
```

Comment:
I'm not sure but I have the feeling this enums should be lowercase. Something similar to what happened with `ManagedResourceStatus` and [the related PR](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/638) (still open).

Reply:
well, the enums is bigger topic and I would like to do an improvement via this jira https://issues.redhat.com/browse/MGDOBR-639.

The current issues with enums on backend are:
- some of them do not follow java convention, i.e. `Car` instead of `CAR`
- some of them support custom serialization, some of them does not

Reply:
:+1: